### PR TITLE
support ftps explicit mode

### PR DESCRIPTION
**Pull request by xwjdsh**

### CODE REVIEW

File: ftp.go [hunk 1]
```diff
@@ -188,13 +203,15 @@ func DialWithContext(ctx context.Context) DialOption {
 }
 
 // DialWithTLS returns a DialOption that configures the ServerConn with specified TLS config
+// the explicit parameter means the client first runs an explicit command ("AUTH TLS")
 //
 // If called together with the DialWithDialFunc option, the DialWithDialFunc function
 // will be used when dialing new connections but regardless of the function,
 // the connection will be treated as a TLS connection.
-func DialWithTLS(tlsConfig *tls.Config) DialOption {
+func DialWithTLS(tlsConfig *tls.Config, explicit ...bool) DialOption {
```
Comment (jlaffaye) [inline ftp.go]: why a variadic of bool and not just one bool ?

Comment (xwjdsh) [reply ftp.go]: In order not to bring breaking changes to the API.

**Comment by weihanchen** [comment]
Can it be merged as it's exactly what I'm looking for

**Comment by ncw** [comment]
@jlaffaye any chance of merging this? I note there is a fork: https://github.com/BenefitsDataTrust/ftp with this commit in it.

**Pull request closed without merging**